### DISCUSSION TITLE
Brouter and no street names in navigation by voice

### DISCUSSION THREAD

**Discussion by r11d11** [devemux86/cruiser#276] [Q&A] [answered]
I use Brouter offline and mapsforge as offline maps. Works very well. But when I start the navigation, I get no streetnames with the text to speech engine. In the settings and routes and turning points I also only see turn right, turn left as text but no street names. When I change to vallhalla or mapquest in routes and service then the street names are spoken. I this an normal issue with brouter or do I have to turn on an option in brouter or cruiser so that the strert names were spoken. I also tried brouter online. Sams issue.

**Comment by 0709wiwiwi** [comment, marked as answer]
Brouter does not support streetnames.

https://github.com/abrensch/brouter/issues/55

**Comment by r11d11** [comment]
Ok, thank you for the quick answer
